DevelopGitHub/GitHubExtension/GitHubExtension.Statistics.WebApi/GrahpBuilder/Builder.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using GitHubExtension.Statistics.WebApi.CommunicationModels;
+using GitHubExtension.Statistics.WebApi.GrahpBuilder.Abstract;
+using GitHubExtension.Statistics.WebApi.Models;
+
+namespace GitHubExtension.Statistics.WebApi.GrahpBuilder
+{
+    public class Builder
```

Comment:
This class use only in statistics. Why i created it ? 
Because, I had a lot of rows code when I create Graph without pattern Builder.
At the moment, I've builder that help me create Graph.